client.go

```diff
@@ -705,10 +706,21 @@ func (c *Client) Deploy(ctx context.Context, path string) (err error) {
 	// Deploy a new or Update the previously-deployed Function
 	c.progressListener.Increment("Deploying function to the cluster")
 	result, err := c.deployer.Deploy(ctx, f)
+
+	var namespace string
```

Comment:
The namespace is already resolved in `c.deployer.Deploy()`, might be better to set it in the `fn.DeploymentResult` and reuse here.

Reply:
Just to clarify, should I add a field for namespace in the DeploymentResult type and populate it when it's resolved in `c.deployer.Deploy()`?